holidays/countries/malaysia.py

```
@@ -803,117 +817,182 @@ class MalaysiaStaticHolidays:
-        2021: (DEC, 3, malaysia_cup_holiday),
-    }
-
-    special_13_public_holidays = {
+    special_01_public_holidays = {
         2018: (
-            (MAY, 17, election_additional_holiday),
-            (MAY, 18, election_additional_holiday),
+            (MAY, 10, additional_holiday),
+            (MAY, 13, additional_holiday),
         ),
     }
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)